packages/language-server/test/completion-items-spec.ts

```diff
@@ -18,6 +18,11 @@ describe("the UI5 language assistant Code Completion Services", () => {
   // Cursor position after selecting the suggestion
   const CURSOR_POSITION = "${0}";
 
+  // Pre-selected snippet text
+  function getSelectedText(text: string): string {
```

Comment:
Perhaps a slightly different name would make this utility act more like a DSL
we use while building our assertions? e.g:

```javascript
{
   newText: `showNoData="${getSelectedText("true")}"`
   newText: `showNoData="${SELECTED_TEXT("true")}"`
}
```
I found the previous version with `CURSOR_POSITION` easier to read.

Reply:
Done